components/atoms/AccordionItem.jsx

```diff
@@ -6,9 +6,15 @@ import charCodes from '../../constants/charCodes';
 
 class AccordionItem extends Component {
     state = {
-        isOpen: false
+        isOpen: this.props.isOpen
     };
 
+    static getDerivedStateFromProps({ isOpen }) {
+        return {
+            isOpen
+        };
+    }
+
```

Comment:
State is not necessary here, is it? It's just reading the props.

Reply:
it's kinda confusing, i was considering suggesting naming it something like "isAllOpen"

Reply:
Ah, I see, so it's more to open other  accordions? But yeah, it seems the `AccordionItem` component doesn't really change the state, it would be safe to just read the props, right?

Reply:
Ah, that makes sense.